app-modules/student-record-manager/src/Filament/Resources/ManageStudentResource.php

```diff
@@ -0,0 +1,191 @@
+                            ->nullable()
+                            ->numeric()
+                            ->minValue(1920)
+                            ->maxValue(now()->addYears(25)->year),
+                    ])
+                    ->columns(3),
+                Section::make('Contact Information')
+                    ->schema([
+                        TextInput::make('email')
+                            ->label('Primary Email')
+                            ->email()
+                            ->maxLength(255),
+                        TextInput::make('email_2')
+                            ->label('Other Email')
+                            ->email()
+                            ->maxLength(255),
+                        TextInput::make('mobile')
+                            ->label('Mobile')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('phone')
+                            ->label('Other Phone')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('address')
+                            ->label('Address')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('address2')
+                            ->label('Apartment/Unit Number')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('address3')
+                            ->label('Additional Address')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('city')
+                            ->label('City')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('state')
+                            ->label('State')
+                            ->string()
+                            ->maxLength(255),
+                        TextInput::make('postal')
+                            ->label('Postal')
+                            ->string()
+                            ->maxLength(255),
+                    ])
+                    ->columns(3),
+                Section::make('Engagement Restrictions')
+                    ->schema([
+                        Radio::make('sms_opt_out')
+                            ->label('SMS Opt Out')
+                            ->boolean(),
+                        Radio::make('email_bounce')
+                            ->label('Email Bounce')
+                            ->boolean(),
+                        Radio::make('dual')
+                            ->label('Dual')
+                            ->boolean(),
+                        Radio::make('ferpa')
+                            ->label('FERPA')
+                            ->boolean(),
```

Comment:
Please use toggles for these and all Radio inputs below. We have always used toggled for bool inputs